### PR TITLE
Initial draft for content

### PR DESCRIPTION
**Pull request by siddharth0248**
## Why are you creating this Pull Request?

- [Adding Datasets or Discoveries](?title=Content%3A%20%3Cname%3E&expand=1&template=content.md)
- [Version Release](?title=Deploy%20vX.X.X&expand=1&template=version_release.md)
- [Other](?expand=1&template=default.md)

### CODE REVIEW

**Comment by slesaad** [comment]
@siddharth0248 a couple of things,
- make all the images less than 500kb, check out the [media guide](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/content/frontmatter/media.md) for how to do that. the images are taking some time to load, heavier files slow down the whole website
- population density dataset isn't working
- EPA methane emissions: add tags?
- the population banner image is pixelated, can we use a better image?
- use sub script and superscripts for the data units

File: datasets/nasa-sedac-population-density.data.mdx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+usage:
+  - url: 'https://github.com/NASA-IMPACT/GHGC_docs/blob/main/cog_transformation_scripts/NASA_GSFC_wetlands_transformation.py'
+    label: Notebook to trasnform netcdf to COG
+    title: 'ODIAC CO2 emissions data trasnsformation notebook'
+  - url: 'https://github.com/NASA-IMPACT/veda-docs/blob/no2-single-item-viz/example-notebooks/no2-single-item-vizualization.ipynb'
+    label: View example notebook
+    title: 'Static view in VEDA documentation'
+media:
+  src: ::file ./gpw-yearly--cover.png
+  alt: aa
```
Comment (slesaad) [inline datasets/nasa-sedac-population-density.data.mdx:14]: 👀

File: datasets/odiac-monthly-2022.data.mdx [hunk 2]
```diff
@@ -63,21 +63,48 @@ layers:
...
+      
+    - **Temporal extent:** 2000/01 - 2021/12    
+    - **Temporal resolution:** Monthly    
+    - **Spatial extent:** Global    
+    - **Spatial resolution:** 1 km x 1 km    
+    - **Data units:** gC m-2 d-1
```
Comment (slesaad) [inline datasets/odiac-monthly-2022.data.mdx:72]: use superscript

File: overrides/components/header-brand/component.tsx [hunk 3]
```diff
@@ -39,7 +39,7 @@ export default function HeaderComponent() {
     <Brand>
       <Link to="/">
         <img src={ghgLogo} />
-        <span>GHG Center</span>
+        <span>GHG Monitoring and Information Center</span>
```
Comment (slesaad) [inline overrides/components/header-brand/component.tsx:42]: Change to US Greenhouse Gas Center?

**Comment by slesaad** [comment]
also, give credits to the photographers for their images

File: datasets/casa-gfed-co2-emission-monthly.data.mdx [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+media:
+  src: ::file ./geos-casa-gfed--cover.jpg
+  alt: aa
+  author:
+    name: Dan Meyers
+thematics:
+  - air-quality
+  - co2
+sources:
+  - NASA
```
Comment (slesaad) [inline datasets/casa-gfed-co2-emission-monthly.data.mdx:21]: should be `nasa` instead. you need to use the `id` of the sources from this yml https://github.com/NASA-IMPACT/veda-config-ghg/blob/add-test-sid/taxonomies.yml

File: datasets/ch4-wetland-monthly.data.mdx [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  src: ::file ./ch4-wetland--cover.jpeg
+  alt: aa
+  author:
+    name: Mark SubbaRao (NASA/GSFC)
+    url: https://svs.gsfc.nasa.gov/5054
+thematics:
+  - air-quality
+  - co2
+sources:
+  - NASA
```
Comment (slesaad) [inline datasets/ch4-wetland-monthly.data.mdx:22]: `nasa`

File: datasets/emit-plume.data.mdx [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+media:
+  src: ::file ./emit-plume--cover.jpg
+  alt: aa
+  author:
+    name: Chris Leboutillier
+thematics:
+  - air-quality
+  - co2
+sources:
+  - NASA
```
Comment (slesaad) [inline datasets/emit-plume.data.mdx:21]: `nasa`

File: datasets/nasa-sedac-population-density.data.mdx [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+  author:
+    name: Columbia University
+thematics:
+  - air-quality
+  - co2
+sources:
+  - NASA
```
Comment (slesaad) [inline datasets/nasa-sedac-population-density.data.mdx:21]: `nasa`

File: datasets/emit-plume.data.mdx [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+media:
+  src: ::file ./emit-plume--cover.jpg
+  alt: emission from industry
+  author:
+    name: Chris Leboutillier
+thematics:
+  - air-quality
+  - ch4
+sources:
+  - NASA
```
Comment (slesaad) [inline datasets/emit-plume.data.mdx:21]:
```suggestion
  - nasa
```

File: datasets/nasa-sedac-population-density.data.mdx [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+    title: 'Static view in VEDA documentation'
+media:
+  src: ::file ./gpw-yearly--cover.png
+  alt: aa
+  author:
+    name: Columbia University
+thematics:
+  - soceco
+sources:
+  - NASA
```
Comment (slesaad) [inline datasets/nasa-sedac-population-density.data.mdx:20]:
```suggestion
  - nasa
```

**Review by slesaad — APPROVED** [review]
Looks good.

**Comment by j08lue** [comment]
This is ready for merging and approved by @siddharth0248 and @slesaad, so all that is needed is that the merge conflicts be resolved.

**Pull request merged**